libs/designer/src/lib/core/utils/parameters/helper.ts

```diff
@@ -294,7 +294,7 @@ function getVisibility(parameter: any) {
 
 function shouldHideInUI(parameter: ResolvedParameter): boolean {
   const visibility = getVisibility(parameter);
-  return parameter?.hideInUI || equals(visibility, 'hideInUI') || equals(visibility, Visibility.Internal);
+  return parameter?.hideInUI || parameter?.schema?.hideInUI || equals(visibility, 'hideInUI') || equals(visibility, Visibility.Internal);
```

Comment:
When do we have hideInUI inside parameter schema?

Reply:
At some point during our importing of the manifest we put the parameter object behind a `schema` prop, in the manifests I'm putting the `hideInUI` prop at the highest level and it still doesn't apply unless I add this schema step.

Reply:
If this resolved parameter is similar to ParameterInfo then there should be method called createParameterInfo in utils/parameters/helper.ts that should be reading this prop corectly and putting it as first class property in parameterinfo.

Reply:
The `createParameterInfo` function gets the `hideInUI` prop from this `shouldHideInUI` function I'm referencing above ^^